Sources/Authenticator/States/AuthenticatorBaseState.swift

```diff
@@ -195,6 +196,18 @@ public class AuthenticatorBaseState: ObservableObject {
             return "authenticator.authError.incorrectCredentials".localized()
         }
 
+        if case .validation(let field, _, _, _) = error {
```

Comment:
Wouldn't it make more sense to not allow to submit if you haven't selected  something? 

This feels like a validation that the view should perform.